metron-platform/metron-common/README.md

```diff
@@ -39,11 +40,17 @@ The following keywords need to be single quote escaped in order to be used in St
 | <= | \> | \>= |
 | ? | \+ | \- |
 | , | \* | / |
+|  | \* | / |
 
 Using parens such as: "foo" : "\<ok\>" requires escaping; "foo": "\'\<ok\>\'"
 
+## Stellar Language Inclusion Checks (`in` and `not in`)
+1. `in` supports string contains. e.g. `'foo' in 'foobar' == true`
+2. `in` supports collection contains. e.g. `'foo' in [ 'foo', 'bar' ] == true`
+3. `in` supports map key contains. e.g. `'foo' in { 'foo' : 5} == true`
+4. `not in` is the negation of the in expression. e.g. `'grok' not in 'foobar' == true`
```

Comment:
Sorry, one last comment. I was a bit curious as to how the expression `'grok' not in 'foobar' == true` would be evaluated by Stellar. I wasn't sure if it would be `'(grok' not in 'foobar') == true` or `'grok' not in ('foobar' == true)`.  Unfortunately when I tried to run a test it said it is not a valid expression. I think this may be an issue in the Stellar grammar. It is probably outside the scope of this ticket, but I thought I should mention it here.